.github/workflows/pr-markdown-link-check.yaml

```diff
@@ -0,0 +1,17 @@
+name: PR Markdown Link Check
+run-name: ${{github.event.pull_request.title}}
+on: [ pull_request ]
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: gaurav-nelson/github-action-markdown-link-check@v1
+        with:
+          use-quiet-mode: 'yes'  
+          use-verbose-mode: 'yes'
+          config-file: '.mlc.config.json'
+          folder-path: '.'
+          max-depth: -1
+          check-modified-files-only: 'yes'
+          base-branch: 'main'
```

Comment:
Using a static `main` branch here is fine if your SDLC revolves around the `main` branch. However, using ` ${{ github.base_ref }}` allows you to use a dynamic one. In this case, the check will be performed based on the branch you are merging the PR into (for example, if you have a release branch, etc...)